apps/cookbook/src/app/examples/alert-example/alert-example.component.ts

```diff
@@ -32,7 +32,7 @@ this.modalController.showAlert(config);`;
       .replace(/"/g, "'");
   }
 
-  private alertClose$ = new Subject();
+  private alertClose$: Subject<void> = new Subject<void>();
```

Comment:
This can pretty much be ignored, as it is just taken from #2462 (RxJS 7 stuff) so we are able to use the user-flows library.